i18n/en.toml

```
@@ -274,6 +274,15 @@ other = "Yes"
 [feedback_no]
 other = "No"
 
+[feedback_ask_for_comment]
+other = "What could make this information more useful?"
```

Maybe this could be worded to sound more optional? For example:
```suggestion
other = "Do you have any suggestions for improvement?"
```